setup.py

```diff
@@ -27,7 +27,7 @@ def read(fname):
     install_requires=(
         'Django>=1.5',
         'easy-thumbnails>=1.0',
-        'django-mptt>=0.6',
+        'django-mptt>=0.6,<0.8',
```

Comment:
can we switch from travis envs to running tox commands the same way as on ``aldryn-events`` or ``aldryn-newsblog`` or ``aldryn-jobs``? I've adjusted ``tox`` setup here https://github.com/divio/django-filer/pull/643